blockscout-ens/bens-proto/proto/bens.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package blockscout.bens.v1;
+
+option go_package = "github.com/blockscout/blockscout-rs/bens";
+
+service DomainsExtractor {
+  rpc SearchAddress(SearchAddressRequest) returns (SearchAddressResponse) {}
+  rpc GetDetailedDomain(GetDetailedDomainRequest) returns (GetDetailedDomainResponse) {}
+}
+
+enum DomainSource {
+  /// https://docs.ens.domains/
+  ENS = 0;
+  // /// https://docs.unstoppabledomains.com/
+  //UNSTOPPABLE_DOMAINS = 1;
+}
+
+message Domain {
+  /// Source of domain name. Now could be only ENS, in future we could add UNSTOPPABLE_DOMAINS
+  DomainSource domain_source = 1;
+  /// Unique id for the domain, also known as nodehash
+  string id = 2;
+  /// The human readable name, if known. Unknown portions replaced with hash in square brackets (eg, foo.[1234].eth)
+  string name = 3;
+  /// Integer representation of labelhash
+  uint64 token_id = 4;
+  /// The account that owns the domain
+  string owner = 5;
+  /// Optinal. Resolved address of this domain
+  optional string resolved_address = 6;
+  /// Optinal. The account that owns the ERC721 NFT for the domain
+  optional string registrant = 7;
+  /// Optinal. Unix timestamp of expiry date. None means never expires
+  optional uint64 expiry_date = 8;
+  /// Unix timestamp of regisration date
+  uint64 registration_date = 9;
+  /// Map chain -> resolved_address. This map will contain `current_chain_id` -> `resovled_address` if `resovled_address` is not None
+  map<string, string> other_address = 10;
```

Comment:
What are the use cases we need this map for?

Reply:
duplicate probably